assets/styles/string-calculator.css

```diff
@@ -0,0 +1,45 @@
+.calculator{
+    background-color: #000;
+    padding: 20px;
+    border-radius: 15px;
+    display: grid;
+    grid-template-rows: 1fr 3fr;
+    max-width: 200px;
+    margin: auto;
+}
+.calc-window{
+    color: #fff;
+    display: flex;
+    align-items: flex-end;
+    justify-content: flex-end;
+    font-size: 30px;
+}
+.elements{
+    color: #fff;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr 1fr;
+    grid-gap: 10px;
+}
+
+span{
+    border: none;
+    cursor: pointer;
+    border-radius: 10px;
+    padding: 10px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: #f17a1e;
+}
+
+#0,#1,#2,#3,#4,#5,#6,#7,#8,#9{
```

Comment:
_Code style_: consider styling these elements using the class name.

Reply:
fixed